src/components/Graphic.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import styled from "@emotion/styled";
+
+const StyledImg = styled.img`
```

Comment:
`StyledGraphic`. `StyledImg` is somewhat opaque since `<img />` tags are applicable to a wide range of components.